mrp_production_note/views/mrp_production_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="mrp_production_view_notes_form" model="ir.ui.view">
+        <field name="name">mrp.production.notes.form</field>
+        <field name="model">mrp.production</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view" />
+        <field name="arch" type="xml">
+            <notebook>
+                <page string="Notes">
```

Comment:
This could be improved for further inheritances:
```suggestion
                <page string="Notes" name="notes">
```
This improvement could be placed in a separated commit, so it could be cherry-picked in previous versions